tests/test_search.py

```diff
@@ -625,6 +635,8 @@
     (TEST_QUERY_21, False, True, 27, 1),  # Accessing 3 elements in test_op_2, plus 9 in test_op_3 (non-flattened)
     (TEST_QUERY_22, False, True,  9, 9),  # Accessing 9 in test_op_3 and checking them against itself
     (TEST_QUERY_23, False, True, 27, 1),  # test_op_3: 9, test_op_1: 3
+    # TODO it fails with errors ' assert False == True' and 'assert 0 == 2', must be 2 matching biosamples
+    (TEST_QUERY_24,  False, True,  2, 2),
```

Comment:
@davidlougheed Hi David, currently the test fails and I don't understand why - the TEST DATA 1 contains two biosamples, each with a procedure where code's label contains 'LABEL' in it. I query for '%label%' so the matching result should return two biosamples but the test returns 0. Maybe you have some idea what's wrong?

Reply:
this test passes now